### ISSUE TITLE
[ACCESSIBILITY] Screen reader for Favorites and Sections on Home screen

### ISSUE THREAD

**Issue by nataliehelregel** [rokwire/illinois-app#4411]
When using a screen reader to access our app-
- Both "Favorites" and "Sections" make a sound but are not read.
- "App Sections" as a header is redundant to the "Sections" toggle above it and should be removed from the design

Next steps would be-
1. Update "Favorites" and "Sections" on the Home screen (circled in green) to be labeled and read by a screen reader.
2. Remove "App Sections" header from the Sections screen of the home page (crossed out in yellow). A mock up of the design is provided in the second screenshot.

![01_App Sections](https://github.com/user-attachments/assets/c9b78453-b157-4465-8992-a7cf7a4f88f8)
![02_App Section Removed](https://github.com/user-attachments/assets/86d83a81-4229-4112-9151-a83213a9e93f)

cc: @acovher2

**Comment by TodorBachvarov** [comment]
@nataliehelregel This should be fixed in version 6.1.13

**Comment by vburgett** [comment]
Thank you, @TodorBachvarov. Can you match the spacing between the image and Academics to match the image Natalie provided above? There should be less space. 

@nataliehelregel 

![Screenshot_20241021-222202.png](https://github.com/user-attachments/assets/66ea890c-b03e-4de1-b407-2d1fb8adfcb6)

**Comment by TodorBachvarov** [comment]
Hi @vburgett, This should be fixed in version 6.1.17

**Comment by acovher2** [comment]
Verified as Fixed.
